misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/reporting/translating/TranslationSupplierBasedTranslatorSpec.kt

```diff
@@ -91,7 +96,9 @@ abstract class TranslationSupplierBasedTranslatorSpec(
         describe("translating a ${Translatable::class.simpleName} to $greatBritain without fallbacks") {
 
             context("no translations provided at all") {
-                val testee = testeeFactory(mock(), greatBritain)
+                val testee = testeeFactory(mockk {
+                    every { get(any(), any()) } returns null
```

Comment:
that does not seem to be right, intended?

Reply:
It actually is intended
mockk is more strict than the older mock
What it means is that we have to supply a specification to methods used by the object,
I provided the general implementation of get method in order for it to pass the tests.

Leaving it only by using empty mockk object will fail tests execution